src/components/Receiving/ReceivingHistory.js

```diff
@@ -271,12 +272,13 @@ class ReceivingHistory extends Component {
                 poLineNumber: <FormattedMessage id="ui-orders.receiving.poLine" />,
                 dateOrdered: <FormattedMessage id="ui-orders.receiving.dateOrdered" />,
                 dateReceived: <FormattedMessage id="ui-orders.receivingHistory.dateReceived" />,
+                comment: <FormattedMessage id="ui-orders.receiving.comment" />,
                 barcode: <FormattedMessage id="ui-orders.receiving.barcode" />,
                 receivingNote: <FormattedMessage id="ui-orders.receivingHistory.note" />,
                 receivingStatus: <FormattedMessage id="ui-orders.receiving.status" />,
               }}
-              columnWidths={{ isChecked: 35 }}
-              visibleColumns={['isChecked', 'title', 'poLineNumber', 'dateOrdered', 'dateReceived', 'barcode', 'receivingNote', 'receivingStatus']}
+              columnWidths={{ isChecked: 35, comment: 253 }}
```

Comment:
Why exactly 253? ))

Reply:
I took it from real HTML, I would prefer to set max-width, but as I understand we can't do it